astropy/coordinates/angles.py

```diff
@@ -550,11 +550,19 @@ def _validate_angles(self, angles=None):
         else:
             limit = u.degree.to(angles.unit, 90.0)
 
-        invalid_angles = (np.any(angles.value < -limit)
-                          or np.any(angles.value > limit))
-        if invalid_angles:
-            raise ValueError('Latitude angle(s) must be within -90 deg <= angle <= 90 deg, '
-                             'got {}'.format(angles.to(u.degree)))
+        if np.any(angles.value < -limit) or np.any(angles.value > limit):
+            with np.printoptions(precision=2):
```

Comment:
I suppose a very simple solution could be to merely put the exception handler into a
`np.printoptions(precision=2, threshold=8)` context – that just wouldn't necessarily identify the guilty values.